s4cmb/tod.py

```diff
@@ -566,8 +630,18 @@ def get_boresightpointing(self):
             # self.HealpixFitsMap.U = self.HealpixFitsMap.U[pix]
 
         self.pointing = Pointing(
-            az_enc=self.scan['azimuth'],
-            el_enc=self.scan['elevation'],
+            az_enc=self.scan['azimuth'], # degrees, size of nts
+            el_enc=self.scan['elevation'], # degrees, size of nts
+            time=self.scan['clock-utc'],
+            value_params=self.hardware.pointing_model.value_params,
+            allowed_params=self.hardware.pointing_model.allowed_params,
+            ut1utc_fn=self.scanning_strategy.ut1utc_fn,
+            lat=lat, ra_src=ra_src, dec_src=dec_src)
+        
+        # boresight pointing systematics
```

Comment:
I would enter this part of code only through an `if` statement conditional on the presence of pointing perturbation injected to save memory